distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column.enso

```diff
@@ -2719,3 +2672,34 @@ Column.from (that:Range) (name:Text="Range") = Column.from_vector name that.to_v
 ## PRIVATE
    Conversion method to a Column from a Vector.
 Column.from (that:Date_Range) (name:Text=that.default_column_name) = Column.from_vector name that.to_vector
+
+## PRIVATE
+   Applies a UnaryOperation to a column.
+
+   Arguments:
+   - column: The column to apply the operation to.
+   - op_name: The name of the operation to apply.
+   - new_name: The name of the column created as the result of this operation.
+apply_unary_operation column:Column op_name:Text new_name:Text|Nothing=Nothing =
+    used_name = new_name.if_nothing (naming_helper.function_name op_name [column])
+    Java_Problems.with_map_operation_problem_aggregator column.name Problem_Behavior.Report_Warning java_problem_aggregator->
+        java_column = UnaryOperation.apply column.java_column op_name used_name java_problem_aggregator
+        if java_column.is_nothing.not then Column.Value java_column else Nothing
+
+## PRIVATE
+   Applies a function to every row in the column.
+
+   Arguments:
+   - column: The column to apply the operation to.
+   - new_name: The name of the column created as the result of this operation.
+   - fallback: A function used if the vectorized operation isn't available.
+     It should never raise dataflow errors.
```

Comment:
This name and comment are no longer valid.

This is not a `fallback` anymore as there is no vectorized operation here.

```suggestion
   - function: A function applied to every element.
     It should never raise dataflow errors.
```